lib/http.go

```diff
@@ -226,7 +226,7 @@ func NewHTTPRequestHandler(inst *Instance, libMethod string) http.HandlerFunc {
 			return
 		}
 
-		if err := DecodeParams(r, &p); err != nil {
+		if err := DecodeParams(r, p); err != nil {
```

Comment:
@ramfox thanks for the hint in https://github.com/qri-io/qri/issues/1758. The fix for actually parsing the params was very needed here.